examples/guestbook-go/v1beta3/guestbook-controller.json

```diff
@@ -0,0 +1,39 @@
+{
+    "kind": "ReplicationController",
+    "apiVersion": "v1beta3",
+    "metadata": {
+        "name": "guestbook",
+        "labels": {
+            "name": "guestbook"
+        }
+    },
+    "spec": {
+        "replicas": 3,
+        "selector": {
+            "name": "guestbook"
+        },
+        "template": {
+            "metadata": {
+                "name": "guestbook",
```

Comment:
That is correct - argument for validating the versioned structs, not the internal.


> On Feb 23, 2015, at 5:33 PM, Brian Grant <notifications@github.com> wrote:
> 
> In examples/guestbook-go/v1beta3/guestbook-controller.json:
> 
> > +    "kind": "ReplicationController",
> > +    "apiVersion": "v1beta3",
> > +    "metadata": {
> > +        "name": "guestbook",
> > +        "labels": {
> > +            "name": "guestbook"
> > +        }
> > +    },
> > +    "spec": {
> > +        "replicas": 3,
> > +        "selector": {
> > +            "name": "guestbook"
> > +        },
> > +        "template": {
> > +            "metadata": {
> > +                "name": "guestbook",
> I think this name is ignored:
> https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/controller/replication_controller.go#L70
> 
> I'm guessing we're not validating that name isn't specified due to legacy v1beta1/2 support.
> 
> Can you confirm, @smarterclayton?
> 
> —
> Reply to this email directly or view it on GitHub.
>